geode-cq/src/distributedTest/java/org/apache/geode/internal/cache/PutAllClientServerDistributedTest.java

```diff
@@ -2459,13 +2401,10 @@ public void testPartialKeyInPRSingleHopWithRedundancy() throws Exception {
     assertThat(newSizeOnClient2).isEqualTo(newSizeOnServer1);
   }
 
-
   /**
    * The purpose of this test is to validate that when two servers of three in a cluster configured
    * with a client doing singlehop, that the client gets afterCreate messages for each entry in the
-   * putall.
-   * Further, we also check that the region size is correct on the remaining server.
-   *
+   * putall. Further, we also check that the region size is correct on the remaining server.
    */
   @Test
   public void testEventIdOutOfOrderInPartitionRegionSingleHop() {
```

Comment:
Unfortunately, I haven't been able to fix the flakiness of testEventIdOutOfOrderInPartitionRegionSingleHop.